src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -172,6 +172,7 @@ load_dh_params(const char *filename) {
   if(filename == NULL) return NULL;
   bio = BIO_new_file(filename, "r");
   if(bio == NULL) return NULL;
+  mtevL(mtev_notice, "Loading DH parameters from %s.\n", filename);
```

Comment:
I still think this should be debugging... if it fails, error.

I can agree that non-normatives (like generating it and saving it) could be considered notices.  But loading the params isn't notable.